src/examples/profile/xml/NIST_SP-800-53_rev5_EXAMPLECONTROLS-baseline_profile.xml

```diff
@@ -1,46 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Modified by the OSCAL 1.0.0 RC2 to OSCAL 1.0.0 conversion XSLT on 2021-06-08T13:57:33.97549-04:00 -->
-<profile xmlns="http://csrc.nist.gov/ns/oscal/1.0"
-         uuid="8742196d-86ba-4e72-a411-28867dab43bb">
+<profile xmlns="http://csrc.nist.gov/ns/oscal/1.0" uuid="c657e7dd-8b8e-4261-891b-caf94e7149dd">
+
    <metadata>
       <title>NIST Special Publication 800-53 Revision 5 LOW IMPACT BASELINE</title>
-      <last-modified>2021-06-08T13:57:33.97549-04:00</last-modified>
-      <version>Final</version>
-      <oscal-version>1.0.0</oscal-version>
+      <last-modified>2022-09-29T18:34:00.00000-04:00</last-modified>
+      <version>Draft 1</version>
+      <oscal-version>1.0.4</oscal-version>
+
       <role id="creator">
          <title>Document Creator</title>
       </role>
       <role id="contact">
          <title>Contact</title>
       </role>
-      <party uuid="f8541f6b-ef0a-4638-98d0-7f214a9b4b28" type="organization">
-         <name>OSCAL Program</name>
-         <link rel="website" href="https://pages.nist.gov/OSCAL/contact/" />
+
+      <party uuid="dc31b097-c02f-4b1d-8b0a-4c728493a5ef" type="organization">
+         <name>GovReady PBC</name>
+         <email-address>gregelin@govready.com</email-address>
+         <address>
+            <addr-line>GovReady PBC</addr-line>
+            <city>Chicago</city>
+            <state>IL</state>
+            <postal-code>60601</postal-code>
+         </address>
       </party>
```

Comment:
OK, so you intend to take ownership of this in the repo and its managament? Last we had discussed this, I thought that was the reason we kept it pointed at "the OSCAL community" and the web address of this git repo on GitHub, respectively.

I am not sure if @david-waltermire-nist has any impressions/concerns with it, but this seems to be flip in intent from our previous chat. Will we talk next week about this?

Reply:
I'm comfortable with ownership identified with NIST or with GovReady. Changing the content was an opportunity to work with the data and the format and see how it looked.

Changing the content made me think about the UUID for GovReady and raised several questions:
- Can GovReady PBC have more than one UUID? What someone in my org creates a different UUID?
- Is the UUID I just a local convenience reference, or am I now responsible for always using the UUID?
- Can other people use that UUID when referring to GovReady PBC? On one hand I want others to use, but not to use GovReady PBC's UUID falsely.
- How would I publish GovReady's reference information of UUID and address information for others to find (e.g., directory services)? What if I update my address -- does my UUID change?
- etc., etc., etc...

Reply:
@gregelin, thanks for your time today. I will want to circle back with Dave on the organizational mechanics of how we handle you owning this with the contact info of the metadata block. Stay tuned, I will update when he returns from leave.

Re your questions below, I realized I had not addressed these during the call. I will answer some of them. When I say "must" the OSCAL information and data models require it, "can" or "could" means we don't restrict it in the models and it is up to you and other developers/community users how they would do it.

> - Can GovReady PBC have more than one UUID? What someone in my org creates a different UUID?

You can have multiple UUIDs. The OSCAL specs do _not_ say one organization must have a UUID in all interactions. An OSCAL-enabled tool _could_ choose to flag that as an error and ask to unify them, or use both. You can see in our examples we give UUIDs to organizations that are actually different units of agencies or specific units of a component bureau (NIST is a part of the Department of Commerce). This flexiblity means you could have multiple UUIDs and tools can decide how they make people and their orgs manage one or more of them.

> - Is the UUID I just a local convenience reference, or am I now responsible for always using the UUID?

As you put it with the two options, a a local convenience reference.

> - Can other people use that UUID when referring to GovReady PBC? On one hand I want others to use, but not to use GovReady PBC's UUID falsely.

Anyone _could_ use the UUID but OSCAL does not specify how an OSCAL-enabled tool must handle that concern as it is really up to how human users interpret it. I understand your point, but different orgs must refer to one another, and UUIDs are to avoid conflicts and allow faster indexing/lookup than inconsistent, free-form names. UUIDs in OSCAL are like the very prevalent use of UUIDs in the Microsoft Windows Installer ecosystem (they call them GUIDs, but it is the same thing). No one centrally manages or communicates GUID for a Microsoft Office or Slack or Adobe Photoshop GUIDs across the ecosystem or often even for their own company.

> - How would I publish GovReady's reference information of UUID and address information for others to find (e.g., directory services)? What if I update my address -- does my UUID change?

OSCAL does not say you must change the UUID. You could change the UUID in an OSCAL document with an OSCAL-enabled tool based on [how significant a change you make to the subject attached to the UUID](https://pages.nist.gov/OSCAL/concepts/identifier-use/#consistency). Maybe for the author of a SSP or their SSP authoring tool, the address of the org does not have a significant impact. What about the address of a data-center where the system described in the SSP is operated and maintained? (Personally, and this is just my opinion and does not confer any strong guidance recommendation from the NIST OSCAL Team: I'd not change the UUID for the former and would for the latter since that is a meaningful change; the former it's the same legel entity and the SSP authoring software and its development org, as a subject, is not meaningfully changed by a HQ address change.)

> - etc., etc., etc...

Let me know if there are/were additional details or nuances I didn't cover in the above answers, or if you have new questions. Hope this helps!